packages/foreman/nodejs-jstz/nodejs-jstz.spec

```diff
@@ -1,63 +1,50 @@
 %global npm_name jstz
+%global enable_tests 1
 
 Name: nodejs-%{npm_name}
 Version: 1.0.7
```

Comment:
I think this should be 1.0.11 based on the [diff file](https://paste.fedoraproject.org/paste/0dp1Gbbia8YniQtSEGHC6Q), lines 67 and 68

Reply:
Good catch -- updated